docs/quickstart.md

```diff
@@ -88,14 +95,13 @@ you can follow the rest of the guide the same way as if not using docker.
 
 ## Preliminaries
 
-We have created a  <a href="https://github.com/fetchai/agents-template" target="_blank">template repo for AEA development</a> here which you can optionally fork and use immediately.
+Ensure, you are in a clean working directory:
 
-Alternatively, create and enter into a new working directory:
+- either you create it manually `mkdir my_aea_projects/ && cd my_aea_projects/`, then add an empty directory called `packages` with the following command `mkdir packages/`,
```

Comment:
Minor: 
This part is also covered in
`Approach 1` in the <a href="../development-setup#approach-1">development setup</a> guide.